api/models/Pokemon.js

```diff
@@ -107,7 +108,11 @@ const attributes = {
   regionId: {},
   consoleRegion: {type: 'string'},
   language: {type: 'string'},
-  _rawPk6: {type: 'string'},
+
+  // TODO: Remove references to _rawPk6 (only used here to avoid needing a migration)
```

Comment:
Put in a ticket number. I don't like TODOs without a ticket number, because they get lost.